### PR TITLE
[kjobctl] Skip kueue labels on child objects.

### PR DESCRIPTION
**Pull request by mbobrovskyi**
#### What type of PR is this?
/kind cleanup

#### What this PR does / why we need it:
We don't need to add kueue labels to child objects like ConfigMap or Service.

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

### CODE REVIEW

**Comment by mimowo** [comment]
cc @mwysokin

File: cmd/experimental/kjobctl/pkg/builder/slurm_builder.go [hunk 1]
```diff
@@ -190,6 +190,26 @@ func (b *slurmBuilder) validateMutuallyExclusiveFlags() error {
 	return nil
 }
 
+func (b *slurmBuilder) buildObjectMeta(templateObjectMeta metav1.ObjectMeta) metav1.ObjectMeta {
+	objectMeta := b.Builder.buildObjectMeta(templateObjectMeta)
+	objectMeta.GenerateName = ""
```
[the remark this answers is not in the document]
Comment (mbobrovskyi) [reply cmd/experimental/kjobctl/pkg/builder/slurm_builder.go:195]: It’s used in other modes. Only Slurm no need it for now.

[the remark this answers is not in the document]
Comment (trasc) [reply cmd/experimental/kjobctl/pkg/builder/slurm_builder.go:195]: You could modify it to take another argument 

```go
buildObjectMeta(templateObjectMeta metav1.ObjectMeta, stritcNaming bool)
```

and populate Name when `stritcNaming == true`.

[the remark this answers is not in the document]
Comment (mbobrovskyi) [reply cmd/experimental/kjobctl/pkg/builder/slurm_builder.go:195]: With this approach, we’ll need to update the other modes as well. This solution for Slurm is temporary until we find another way to use GenerateName.

[the remark this answers is not in the document]
Comment (mbobrovskyi) [reply cmd/experimental/kjobctl/pkg/builder/slurm_builder.go]: I reviewed the code to find a solution for using GenerateName. Unfortunately, it’s currently not possible without an operator, as this name is used in many places.

**Review by mwysokin — APPROVED** [review]
LGTM 🖖

**Pull request merged**